♻️  Introduce Document Lifecycle into Learn Section (#486)
livingdocsIO/documentation: content/reference-docs/server-extensions/server-hooks.md

## Publication Hooks

> With publication hooks you can influence the [`Document Publication Lifecycle`]({{< ref "/reference-docs/server-extensions/server-events#available-events" >}}) for example by changing content before the publication.

Reviewer: ```suggestion
With publication hooks you can influence the [`Document Publication Lifecycle`]({{< ref "/reference-docs/server-extensions/server-events#available-events" >}}). E.g. change the content before a publish.
```
